src/components/review-card-ratings-only.js

```diff
@@ -2,55 +2,74 @@
-                    </Grid.Column>
-                  </Grid.Row>
-                </Grid>
-
-                <Card.Meta className='course-review-card-star-rating'>
-                  Overall: <Rating icon='star' defaultRating={overallRating} maxRating={5} disabled />
+            <Grid.Row columns={2} className='review-row'>
+              <Grid.Column width={8}>
+                <Card.Header><h3>{reviewTitle}</h3></Card.Header>
+                <Card.Meta>
+                  Overall:
+                  <Rating icon='star' rating={overallRating} maxRating={5} disabled />
+                  <div>
+                    Term taken: {termTaken}
+                  </div>
                 </Card.Meta>
-
-                <div>
-                  <label>Usefulness</label>
-                  <Progress value={usefulProgress} total='5' progress='ratio' size='small' color='blue' />
-                </div>
-                <div>
-                  <label>Workload</label>
-                  <Progress value={workloadProgress} total='5' progress='ratio' size='small' color='blue' />
-                </div>
-                <div>
-                  <label>Enjoyment</label>
-                  <Progress value={enjoymentProgress} total='5' progress='ratio' size='small' color='blue' />
-                </div>
-                <div>
-                  <label>Difficulty</label>
-                  <Progress value={difficultyProgress} total='5' progress='ratio' size='small' color='blue' />
-                </div>
+              </Grid.Column>
+              <Grid.Column width={8}>
+                <div className="Date" style={{ textAlign: 'right' }}>{reviewDate}</div>
+                <Card.Meta className='reviewCardAuthor'>
+                  {author}
+                </Card.Meta>
+              </Grid.Column>
+            </Grid.Row>
+            <Grid.Row columns={4} className='review-row'>
+              <Grid.Column width={3}>
+                <Card.Header>Usefulness</Card.Header>
+                <Card.Header>Workload</Card.Header>
+              </Grid.Column>
+              <Grid.Column width={5}>
+                <ReviewRating
+                  rating={usefulProgress}
+                  icon='circle'
+                  size='small'
+                  captions=''
+                />
+                <ReviewRating
+                  rating={workloadProgress}
+                  icon='circle'
+                  size='small'
+                  captions=''
+                />
+              </Grid.Column>
+              <Grid.Column width={3}>
+                <Card.Header>Enjoyment</Card.Header>
+                <Card.Header>Difficulty</Card.Header>
+              </Grid.Column>
+              <Grid.Column width={5}>
+                <ReviewRating
+                  rating={enjoymentProgress}
+                  icon='circle'
+                  size='small'
+                  captions=''
```

Comment:
can remove `captions=''` on each line since you're not using it anyways

Reply:
when i remove it, it comes up with an error saying it can't read undefined property?